src/Akka.Streams.Kafka.Tests/Dsl/ControlSpec.cs

```diff
@@ -0,0 +1,81 @@
+using System;
+using System.Threading.Tasks;
+using Akka.Streams.Kafka.Helpers;
+using Akka.Util;
+using Xunit;
+
+namespace Akka.Streams.Kafka.Tests.Dsl;
+
+public class ControlSpec
```

Comment:
Verifies that the `IControl` and `DrainingControl` `Task` composition work and make sense.